04-store-api/starter/controllers/products.js

```diff
@@ -0,0 +1,67 @@
+const Product = require('../models/product');
+
+const getAllProductsStatic = async (req, res) => {
+  const products = await Product.find({ price: { $gt: 30 } })
+    .select('name price')
+    .limit(10)
+    .skip(5);
+  res.status(200).json({ products, nbHits: products.length });
+};
+
+const getAllProducts = async (req, res) => {
+  const { featured, company, name, sort, fields, numericFilters } = req.query;
+  const queryObject = {};
+  if (featured) {
+    queryObject.featured = featured === 'true' ? true : false;
```

Comment:
How can we simplify this expression? Writing "true" or "false" in your boolean logic is generally something we can avoid.

We call things like that a "code smell," if you see it, you just know it doesn't smell right, and if you take a closer look, there's definitely a way we can make this better!

Reply:
we can use Boolean constructor like this ```queryObject.featured = Boolean(featured)```
OR double NOT ```queryObject.featured = !!featured```